CONTRIBUTING.md

```diff
@@ -51,6 +53,24 @@ following terms in the code, messages, or comments. Note the alternatives we wil
    * controller, host / device, worker, proxy
    * director / performer
 
+#### Git Commits
+
+##### Rules for Git Commit Message Style
+- Separate subject from the body with a blank line
+- Use past tense verbs
```

Comment:
I think present tense verbs is what we used to recommend? E.g.:
`Add new hashing method` instead of `Added new hashing method`.
It helps with reducing the character limit